README.rst

```diff
@@ -68,32 +69,33 @@ To find where OMERO.figure has been installed using pip, run:
 
     $ pip show omero-figure
 
-The command will display the absolute path to the directory where the application is installed e.g. ``~/<virtualenv_name>/lib/python2.7/site-packages``. Go to that directory. 
+The command will display the absolute path to the directory where the application is installed e.g. ``~/<virtualenv_name>/lib/python3.6/site-packages``. Go to that directory.
 
-*Option 1*: Connect to the OMERO server and upload script via the CLI. It is important to be in the correct directory when uploading so that the script is uploaded with the full path: ``omero/figure_scripts/Figure_To_Pdf.py``:
+*Option 1*: Connect to the OMERO server and upload the script via the CLI. It is important to be in the correct directory when uploading so that the script is uploaded with the full path: ``omero/figure_scripts/Figure_To_Pdf.py``:
 
 ::
 
     $ cd omero_figure/scripts
-    $ path/to/OMERO.server/bin/omero script upload omero/figure_scripts/Figure_To_Pdf.py --official
+    $ omero script upload omero/figure_scripts/Figure_To_Pdf.py --official
 
 *Option 2*: Alternatively, before starting the OMERO.server, copy the script from the figure install
 ``/omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py`` to the OMERO.server ``path/to/OMERO.server/lib/scripts/omero/figure_scripts``. Then restart the OMERO.server.
 
 Now install the script's dependencies:
 
 
-* Install `reportlab <https://bitbucket.org/rptlab/reportlab>`_ PDF python package from distribution packages. For example, install on CentOS 7:
+* Install `reportlab <https://bitbucket.org/rptlab/reportlab>`_ PDF python package from distribution packages.
```

Comment:
``python package from distribution packages`` is a bit heavy
Remove ``from distribution packages``.